vnext/local-cli/runWindows/utils/msbuildtools.js

```diff
@@ -67,9 +67,10 @@ class MSBuildTools {
     if (this.version === '16.0') {
       args.push('/p:PlatformToolset=v141');
       args.push('/p:VisualStudioVersion=16.0');
-      args.push(
-        '/p:VCTargetsPath=C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\MSBuild\\Microsoft\\VC\\v150\\',
```

Comment:
>'/p:VCTargetsPath=C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\MSBuild\\Microsoft\\VC\\v150\\', [](start = 8, length = 118)

Commenting this out fixes the build on local machines, but breaks the build in the CI loop.  We clearly see two different behaviors.  @Julio Cesar Rocha any ideas?  It seems to me this shouldn't be necessary.  We should figure out how to make the CI loop happy without it.